scalyr_agent/platform_windows.py

```diff
@@ -125,6 +125,8 @@
 _SERVICE_DESCRIPTION_ = "Collects logs and metrics and forwards them to Scalyr.com"
 # A custom control message that is used to signal the agent should generate a detailed status report.
 _SERVICE_CONTROL_DETAILED_REPORT_ = win32service.SERVICE_USER_DEFINED_CONTROL - 1
+# A custom control message that is used to signal the agent should generate a health check report.
+_SERVICE_CONTROL_HEALTH_CHECK_ = win32service.SERVICE_USER_DEFINED_CONTROL - 2
```

Comment:
I don't think you should need this anymore.  We should just use the normal status -v signal.